.gitignore

```
@@ -3,3 +3,5 @@
 
 # Ignore Gradle build output directory
 build
+.idea/
+**/*.jar
```

```suggestion
**/*.jar
**/*.sh
```

I discovered that the bash scrip example can bring some confusion! Whta do you think of this change?

---

Yes, I think this would be a good change, Change required [here](https://github.com/clightning4j/btcli4j/blob/master/.gitignore) also: